packages/admin-app/src/pages/ElectionListPage.tsx

```diff
@@ -1,6 +1,115 @@
-import { Grid } from '@material-ui/core';
-import React from 'react';
+import { ApiClientFactory, Election } from '@electionguard/api-client';
+import { Grid, Container, makeStyles } from '@material-ui/core';
+import { DataGrid, GridColDef } from '@material-ui/data-grid';
+import React, { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import FilterToolbar from '../components/FilterToolbar';
+import GoHomeButton from '../components/GoHomeButton';
+import InternationalText from '../components/InternationalText';
 
-export const ElectionListPage: React.FC = () => <Grid>Election List</Grid>;
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 0.75,
+    },
+    title: {
+        fontSize: 40,
+        textAlign: 'center',
+        display: 'block',
+        paddingBottom: theme.spacing(2),
+    },
+    content: {
+        paddingTop: theme.spacing(4),
+        paddingBottom: theme.spacing(4),
+    },
+    grid: {
+        minHeight: 200,
+    },
+    electionState: {
+        textTransform: 'lowercase',
+    },
+    navArea: {
+        paddingBottom: theme.spacing(4),
+        textAlign: 'center',
+    },
+}));
+
+export const ElectionListPage: React.FC = () => {
+    const initialElections: Election[] = [];
+    const [elections, setElections] = useState(initialElections);
+    const [textResult, setTextResult] = useState('');
+    const [pageSize, setPageSize] = React.useState(10);
+    const classes = useStyles();
+    const intl = useIntl();
+
+    const columns = (): GridColDef[] => [
```

Comment:
You'll notice no manifest or context.  I could be convinced to add those in a separate PR, just let me know.